src/server/worker.rs

```diff
@@ -16,6 +16,7 @@ bitflags::bitflags! {
         // There is no "waiting" task for resources of this worker
         // Therefore this worker should not cause balancing even it is underloaded
         const PARKED = 0b00000001;
+        const STOPPING = 0b00000010;
```

Comment:
Maybe add a comment here? `STOPPING` sounds like it's in the process of being stopped, but it's unclear why and what caused it.